File: tourneyAPI/Models/Entities/Characters/ToonLink.cs


namespace Entities;

using Enums;


// Defines the competitive profile metadata for this playable character.
public class ToonLink : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public ToonLink()
    {
        Id = CharacterId.ToonLink;
        characterName = CharacterName.TOON_LINK;
        archetype = Archetype.TURTLE;
        fallSpeed = FallSpeed.FLOATY;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.A;
    }

}

File: tourneyClient/src/models/entities/Characters/ToonLink.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const ToonLink: Character =
{
    id: CharacterId.ToonLink,
    characterName: CharacterName.TOON_LINK,
    archetype: Archetype.TURTLE,
    fallSpeed: FallSpeed.FLOATY,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.A
}

export default ToonLink; 